lgc/builder/BuilderImpl.cpp

```diff
@@ -159,7 +158,6 @@ Value *BuilderImplBase::CreateIntegerDotProduct(Value *vector1, Value *vector2,
         scalar = CreateIntrinsic(intrinsicDot2, {}, {input1, input2, accumulator, clamp}, nullptr, instName);
       } else {
         Value *intermediateRes = getInt32(0);
-        auto indices = ArrayRef<int>({0, 1});
```

Comment:
Can you just make it a stack variable to avoid duplication? E.g., `std::array<int, 2> indices = {0, 1}`;
Also everywhere else.

Reply:
Thanks. Done.